packages/expo-calendar/src/Calendar.ts

```diff
@@ -243,7 +243,7 @@ export async function getEventAsync(
   }
 }
 
-export async function createEventAsync(calendarId: string, details: OptionalKeys<Event> = {}): Promise<string> {
+export async function createEventAsync(calendarId: string, { id, ...details }: OptionalKeys<Event> = {}): Promise<string> {
```

Comment:
I have a quest for you 😂 Please, in separate commit, remove type `OptionalKeys` and replace it with TypeScript default `Partial` built-in type 😜